site/en/blog/new-in-devtools-91/index.md

```diff
@@ -117,6 +117,8 @@ Chromium issue: [1076198](https://crbug.com/1076198)
 ### Support for the `accent-color` keyword {: #accent-color }
 The Styles pane's autocomplete UI now detects the [`accent-color`](https://drafts.csswg.org/css-ui-4/#widget-accent) CSS keyword, which allows web developers to specify the accent color for UI controls (e.g. checkbox, radio button) generated by the element.
 
+The `accent-color` CSS property is currently experimental. Please enable `chrome://flags/#enable-experimental-web-platform-features` to test it.
```

Comment:
Is it worth linking "currently experimental" to this URL, so people can find out its status? https://chromestatus.com/features/4752739957473280

Reply:
Oh, yes! Thanks.